modules/network-observability-operator-install.adoc

```diff
@@ -27,7 +27,8 @@ You can install the Network Observability Operator using the {product-title} web
 * *loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`.
 * *loki.authToken*: Select the `FORWARD` value.
 * *tls.enable*: Verify that the box is checked so it is enabled.
-. Click *Create*.
+* *statusTls*: The `enable` value is false by default. Select the box to enable the selection. You must manually complete the `caCert` and `userCert` fields.
```

Comment:
You dont need to manually enter the values. It has default values defined based on the lokiStack name being `loki`. But I dont know the right wording for that statement.
cc @jpinsonneau

Reply:
+1 default values should be fine now :smile_cat: